test/e2e/Tests/Tests/HelloCitiesTest.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Net;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Azure.Durable.Tests.DotnetIsolatedE2E
+{
+    [Collection(Constants.FunctionAppCollectionName )]
+    public class HttpEndToEndTests
+    {
+        private readonly FunctionAppFixture _fixture;
+
+        public HttpEndToEndTests(FunctionAppFixture fixture, ITestOutputHelper testOutputHelper)
+        {
+            _fixture = fixture;
+            _fixture.TestLogs.UseTestLogger(testOutputHelper);
+        }
+
+        [Theory]
+        [InlineData("HelloCities_HttpStart", "", HttpStatusCode.Accepted, "")]
+        public async Task HttpTriggerTests(string functionName, string queryString, HttpStatusCode expectedStatusCode, string expectedMessage)
+        {
+            HttpResponseMessage response = await HttpHelpers.InvokeHttpTrigger(functionName, queryString);
+            string actualMessage = await response.Content.ReadAsStringAsync();
+
+            Assert.Equal(expectedStatusCode, response.StatusCode);
+
+            if (!string.IsNullOrEmpty(expectedMessage))
+            {
+                Assert.False(string.IsNullOrEmpty(actualMessage));
+                //Assert.Contains(expectedMessage, actualMessage);
```

Comment:
Why is this assert commented out?

Reply:
Carryover from the worker test code this code was derived from. Since the response here is a set of API URLs, we don't do any validation yet. Need to write a set of Durable test methods to parse out the StatusQueryGetAPI URL, invoke until orchestration completion, and parse the "actual" function output as a future work item. 
Deleting this line